packages/web3-react/src/hooks/useConnectorInfo.ts

```diff
@@ -78,7 +73,6 @@ export const useConnectorInfo = (): ConnectorInfo => {
     if (isDappBrowser) return undefined;
 
     if (isGnosis) return PROVIDER_NAMES.GNOSIS;
-    if (isLedger) return PROVIDER_NAMES.LEDGER;
```

Comment:
Is this used internally or externally? Maybe we can proxy wagmi connectors state to retain backward compability.

Reply:
There is no point in using `@reef-knot/web3-react` directly, not as part of `reef-knot` or without `@reef-knot/connect-wallet-modal`. In this case nothing breaks, reef-knot still offers ledger connection, but based on wagmi.
`@reef-knot/web3-react` will be deprecated soon and removed from reef-knot and I believe we don't really need to spend resources to maintain it.